lib/conjur/api/variables.rb

```diff
@@ -105,6 +105,17 @@ def variable_values(varlist)
       end
     end
 
+    # Fetch all visible variables that expire within the given
+    # interval (expressed as an ISO8601 duration). If no duration is
+    # specifed, all variables that are set to expire will be returned.
+    #
+    # param [String] duration ISO8601 duration 
```

Comment:
I'm trying to avoid the ambiguity of using Time on the client. As above, I can change this method to take an argument that supports #to_i to specify the seconds in the duration.